avro-console-scripts/src/assembly/resources.xml

```diff
@@ -0,0 +1,17 @@
+<assembly>
+  <id>resources</id>
```

Comment:
FYI the id of the assembly is what gets used as the artifact classifier in Nexus